### PR TITLE
Added Generic.Arrays.DisallowLongArraySyntax.Found rule

### PR DESCRIPTION
**Pull request by slischka**

### CODE REVIEW

File: src/ruleset-base.xml [hunk 1]
```diff
@@ -42,6 +42,9 @@
 		<exclude name="Squiz.Arrays.ArrayDeclaration.NoComma"/>
 		<exclude name="Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast"/>
 	</rule>
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
+		<type>warning</type>
```
Comment (mabar) [inline src/ruleset-base.xml]: Is there any reason to lower severity from error to warning?

**Comment by slischka** [comment]
No, should I remove it?

File: src/ruleset-base.xml (hunk 1 above)
Comment (slischka) [reply src/ruleset-base.xml]: Updated

**Comment by mabar** [comment]
Lol, Git did not like my force push much :'D
Fixed by https://github.com/orisai/coding-standard-php/commit/eee55af9929520919efcbbda7ea4ee3ab220b1aa, thanks

**Pull request closed without merging**